functions/v2/helloGCS/test/unit.test.js

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// [START functions_cloudevent_storage_unit_test]
+
```

Comment:
nit: consider removing extra lines 16 and 57, this will add spaces in the docs. Approving either way.